language/parser.py

```diff
@@ -565,14 +552,13 @@
         grammar.unqualified_identifier,
         Opt("with", grammar.description_aspect),
         "is",
-        Or(grammar.null_state_body, grammar.state_body),
+        Or(grammar.state_body),
```

Comment:
Couldn't the `Or(...)` be removed altogether?

Reply:
You are right. Fixed.